Ship_Game/Audio/AudioConfig.cs

```diff
@@ -222,7 +224,7 @@ public void Update()
         // remove disposed handles
         lock (TrackedInstances)
         {
-            for (int i = 0; i < TrackedInstances.Count; i++)
+            for (int i = 0; i < TrackedInstances.Count; ++i)
```

Comment:
Is there any purpose of ++i? Or it can be TrackedInstances.Count + 1?

Reply:
Its a shortcut for  `i++`
also `++i` and `i++` differ

Reply:
It's called the idiomatic for loop. You should simply memorize how this is done. It is always the same everywhere.

```cs
for (int i = 0; i < array.Length; ++i)
   Log.Info(array[i].ToString()); 
```

Reply:
I thought i++ is kinda more usual

Reply:
It doesn't make a difference in the big picture.

But if you really care about the semantics
++i means pre-increment
i++ means post-increment